_sass/footer.scss

```diff
@@ -0,0 +1,74 @@
+.footer-col-wrapper {
+  max-width: -webkit-calc(#{$content-width} - (#{$spacing-unit} * 2));
+  max-width:         calc(#{$content-width} - (#{$spacing-unit} * 2));
+  margin-right: auto;
+  margin-left: auto;
+  // padding-right: $spacing-unit;
+  // padding-left: $spacing-unit;
+  @extend %clearfix;
+  color: $color-lightest;
+
+  @include media-query($on-laptop) {
+    max-width: -webkit-calc(#{$content-width} - (#{$spacing-unit}));
+    max-width:         calc(#{$content-width} - (#{$spacing-unit}));
+    padding-right: $spacing-unit / 2;
+    padding-left: $spacing-unit / 2;
+  }
+
+  @include media-query($on-palm) {
+    .footer-col {
+      float: none;
+      width: -webkit-calc(100% - (#{$spacing-unit} / 2));
+      width:         calc(100% - (#{$spacing-unit} / 2));
+    }
+
+    .SIL-logo {
+      height: 63px;
+      width: 130px;
+      padding: 0 5px 0 0;
+    }
+
+    .govt-logo {
+      height: 17px;
+      width: 170px;
+    }
+  }
+}
+
+.SIL-logo {
+  height: 68px;
+  width: 163px;
+  padding: 0 10px 0 0;
+}
+
+.govt-logo {
+  height: 21px;
+  width: 200px;
+  padding: 0 20px 10px 0;
+}
+
+.cc-logo {
+  width: 85px;
```

Comment:
Again, this is very specific... maybe you could achieve what you want by giving the parent element  `display: flex`